.ci_support/linux_64_blas_implgenericc_compiler_version10cuda_compilernvcccuda_compiler_version11.2cxx_compiler_version10.yaml

```diff
@@ -53,7 +53,7 @@ python:
 - 3.8.* *_cpython
 - 3.9.* *_cpython
 pytorch:
-- '2.0'
+- '2.1'
```

Comment:
Might the previous builds for `2.1.0` be compromised?

Reply:
I think this pin is somewhat ignored. This funny thing is happening in rendering this repo that is odd with the PyTorch pinning

Reply:
I find the behaviour of conda build misleading.